containers/keycloak-ers/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/caarlos0/env"
+	"github.com/opentdf/v2/keycloak-ers/handlers"
+
+	"go.uber.org/zap"
+
+	"github.com/virtru/oteltracer"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+)
+
+var cfg EnvConfig
+var svcName = "keycloak-ers"
+
+type EnvConfig struct {
+	KeycloakUrl          string `env:"KEYCLOAK_URL" envDefault:"http://localhost:8080"`
+	KeycloakRealm        string `env:"KEYCLOAK_REALM" envDefault:"tdf"`
+	KeycloakClientId     string `env:"KEYCLOAK_CLIENT_ID" envDefault:"tdf-entity-resolution-service"`
+	KeycloakClientSecret string `env:"KEYCLOAK_CLIENT_SECRET"`
+	KeycloakAuthPath     bool   `env:"KEYCLOAK_AUTH_PATH" envDefault:"false"`
```

Comment:
```suggestion
	KeycloakAuthPath     bool   `env:"KEYCLOAK_AUTH_PATH" envDefault:"false"`
```

I don't think this is necessary - the "Keycloak" part of the auth path can be built/hardcoded with only the Realm as a variable.

The OIDC part of the auth path doesn't change.

Reply:
Ah nevermind I see - the non-OIDC keycloak prefix is different in 18 - got it.